### PR TITLE
quick fix to home

### PR DESCRIPTION
**Pull request by larajanecka**

### CODE REVIEW

File: src/js/pages/attendee/Home.jsx [hunk 1]
```diff
@@ -184,10 +184,10 @@ export default class Featured extends React.Component {
         "link" : branding.package,
       };
 
-      if (preferences.map) {
+      if (preferences.map && branding.map) {
         mapCard = <Card type="download" item={map} buttonStyle={secondaryText}/>;
       }
-      if (preferences.package) {
+      if (preferences.package && branding.map) {
```
Comment (schneider-clarisse) [inline src/js/pages/attendee/Home.jsx]: branding.package

**Review by schneider-clarisse — CHANGES_REQUESTED** [review]

File: src/js/pages/attendee/Home.jsx (hunk 1 above)
Comment (larajanecka) [reply src/js/pages/attendee/Home.jsx]: lul

**Review by schneider-clarisse — APPROVED** [review]
lul

**Comment by ericljpemberton** [comment]
:+1:

**Pull request merged**